.github/CODEOWNERS

```
@@ -8,7 +8,7 @@
 #
 #####################################################
 
-*.md @signalfx/docs
-*.rst @signalfx/docs
-docs/ @signalfx/docs
-README* @signalfx/docs @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers
+*.md @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/gdi-data-collection-maintainers
```

The maintainers are a subset of the approvers so both teams shouldn't be necessary:

```suggestion
*.md @signalfx/docs @signalfx/gdi-data-collection-approvers
```

etc...